android/IotSub.java

```diff
@@ -0,0 +1,10 @@
+Channel channel = ably.channels.get("device:rocket:1");
```

Comment:
This snippet needs to use an MQTT client not an Ably client (see the [equivalent JavaScript snippet](https://github.com/ably/homepage-snippets/blob/main/javascript/iot-sub.js))